assets/style/main.css

```diff
@@ -476,14 +484,56 @@ header {
 }
 
 @media (max-width: 768px) {
+  .popupmenu-container {
+    width: 100vw;
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(96, 112, 255, 0.95);
+    mix-blend-mode: multiply;
+    padding-left: 10px;
+    padding-top: 60px;
+    border: white 1px solid;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: stretch;
+  }
+
```

Comment:
- [ ] Hi @osmanwako, Great  effort working on the mobile menu of your portfolio project, But i noticed somethings that needs attention to work on. when the hamburger is click and the menu displayed i notice some extra space with different color my the bottom and right hand side. I would kindly suggest you change the width and height of the container from `vw`,`vh` to `%` in order to remove the extra spaces.
- [ ] Also, in the Figma design when the mobile menu is displayed, the main page at the background becomes `blur` but in your own case the main page is not blur, kindly check through your styling and try to implement the blurring of the main page when the mobile menu is displayed.


| Your Design 👇   | Figma Design 👇  |
| ------------- | ------------- |
| ![Osman-mob](https://user-images.githubusercontent.com/78634841/223665527-d8ce79af-bbb0-446d-89cf-6fe55bc90d24.jpg)  | ![mob-orig](https://user-images.githubusercontent.com/78634841/223665610-e40e7797-b8f4-40f4-bf85-52ccbb2f6400.png)  |